src/fake_bpy_module/support.py

```diff
@@ -1,40 +1,55 @@
 from typing import List
 
-SUPPORTED_TARGET: List[str] = [
-    "blender",
-    "upbge"
-]
+SUPPORTED_TARGET: List[str] = ["blender", "upbge"]
 
-SUPPORTED_STYLE_FORMAT: List[str] = [
-    "none",
-    "yapf",
-    "ruff"
-]
+SUPPORTED_STYLE_FORMAT: List[str] = ["none", "yapf", "ruff"]
 
 SUPPORTED_MOD_BLENDER_VERSION: List[str] = [
-    "2.78", "2.79",
-    "2.80", "2.81", "2.82", "2.83",
-    "2.90", "2.91", "2.92", "2.93",
-    "3.0", "3.1", "3.2", "3.3", "3.4", "3.5", "3.6",
-    "4.0", "4.1",
-    "latest"
+    "2.78",
+    "2.79",
+    "2.80",
+    "2.81",
+    "2.82",
+    "2.83",
+    "2.90",
+    "2.91",
+    "2.92",
+    "2.93",
+    "3.0",
+    "3.1",
+    "3.2",
+    "3.3",
+    "3.4",
+    "3.5",
+    "3.6",
+    "4.0",
+    "4.1",
+    "latest",
```

Comment:
Also need `fmt: skip` here to keep the original format.

Reply:
Done